cmd/hhfctl/main.go

```diff
@@ -392,6 +392,8 @@ func main() {
 					{
 						Name:  "get",
 						Usage: "Get connections",
+						ArgsUsage: "<type>",
+						Description: "Available types: fabric, external, etc", // TODO: get full list from model
```

Comment:
It currently only have a custom behaviour for management, fabric, vpc-loopback. Everything else is just kubectl get conn, so, you can explicitly list only those 3.

https://github.com/githedgehog/fabric/blob/master/pkg/hhfctl/connection.go